src/components/Menu/MenuItem/MenuItemButton/MenuItemButton.tsx

```diff
@@ -46,7 +49,11 @@ export const MenuItemButton: FC<MenuItemButtonProps> = ({
 
   return (
     <button
```

Comment:
This is an invalid pattern, Suggest using a conditional wrapper or separate elements. Wrapping a `<button>` in a `<button>`   is not accessible.